app/components/detail-display.hbs

```diff
@@ -28,27 +28,31 @@
 </h4>
 <ol data-test-attack-detail-list>
   {{#each @attackDetailsList as |attackDetails index|}}
-  <li>
-    Attack
-    {{#if attackDetails.hit }}
-    inflicted {{attackDetails.damage}} damage
-    {{/if}}
-    {{#unless attackDetails.hit}}
-    missed
-    {{/unless}}
-    with an attack roll of {{attackDetails.roll}}
+
+  {{#if attackDetails.hit }}
+  <hit>
+    Attack roll: {{attackDetails.roll}}
     {{#if attackDetails.crit}} (CRIT!){{/if}}
-    {{#if attackDetails.nat1}} (NAT 1!){{/if}}
+
     <ul data-test-damage-detail-list={{index}}>
       {{#each attackDetails.damageDetails as |damage|}}
       <li>
-        {{damage.label}}: {{damage.roll}} damage
+        {{damage.label}}: {{damage.roll}} damage inflicted
```

Comment:
Once the display is updated to display the actual damage dice on hover, I'll probably remove "inflicted"; for now, it is helpful to clarify that the target's resistance or vulnerability has already been taken into account.